execexam/main.py

```diff
@@ -86,6 +86,7 @@ def run(  # noqa: PLR0913, PLR0915
         litellm_thread.start()
     # add the project directory to the system path
     sys.path.append(str(project))
+    print("This is the correct execexam! You did it! You made it local. You are the best!")
```

Comment:
consider the rich command `console.print` instead of python `print`

Reply:
Yes, please note that we should not use standard `print` statements in the code. It is also very important to point out that _this specific print_ should not be in the final version of the program as it was only there for debugging purposes.

Also, @Coltin2121 please make sure that you use correct commit messages that follow the Conventional Commits standard.